packages/opencensus-core/src/stats/view.ts

```diff
@@ -59,6 +62,8 @@ export class BaseView implements View {
   endTime: number;
   /** true if the view was registered */
   registered = false;
+  /** An object to log information to */
+  private logger: loggerTypes.Logger = logger.logger();
```

Comment:
Should this be hardcoded? I've noticed in other classes in this repo the pattern seems to be to have a default logger but allow passing one in as an override.

Reply:
Fixed for current class, added [issue 177](https://github.com/census-instrumentation/opencensus-node/issues/177) for improve other parts